### PR TITLE
chore: add alerts and dashboard to existing f5 quickstart

### PR DESCRIPTION
**Pull request by thezackm**
# Summary
adding alerts and dashboard tested in production account

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/f5/f5-big-ip.json [hunk 1]
```diff
@@ -0,0 +1,1184 @@
...
+              "row": 1,
+              "width": 3,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Overview\n[f5 BIG-IP](https://www.f5.com/products/big-ip-services) is a set of application delivery products that work together to ensure high availability, improved performance, application security, and access control.\n- - -\n### About this Dashboard\n- For more information on how to instrument your f5 environment, visit the documentation [here](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/f5-monitoring-integration/#install)."
```
Comment (zstix) [inline dashboards/f5/f5-big-ip.json:22]: If possible, can we reference this image as a data url or host it on a server that new relic owns?

If we went the data url route, this is what we would use in place of the icon path.

```
data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iMzJweCIgaGVpZ2h0PSIzMnB4IiB2aWV3Qm94PSIwIDAgMzIgMzIiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgICA8IS0tIEdlbmVyYXRvcjogU2tldGNoIDU1LjIgKDc4MTgxKSAtIGh0dHBzOi8vc2tldGNoYXBwLmNvbSAtLT4KICAgIDx0aXRsZT5Qcm92aWRlciAvIEY1PC90aXRsZT4KICAgIDxkZXNjPkNyZWF0ZWQgd2l0aCBTa2V0Y2guPC9kZXNjPgogICAgPGcgaWQ9IlByb3ZpZGVyLS8tRjUiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPgogICAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMuMDAwMDAwLCAzLjAwMDAwMCkiIGZpbGw9IiNERjA4MzIiIGlkPSJGNSI+CiAgICAgICAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEzLjAwMDAwMCwgMTMuMDAwMDAwKSBzY2FsZSgtMSwgMSkgcm90YXRlKC0xODAuMDAwMDAwKSB0cmFuc2xhdGUoLTEzLjAwMDAwMCwgLTEzLjAwMDAwMCkgdHJhbnNsYXRlKDAuMDAwMDAwLCAtMC4wMDAwMDApIj4KICAgICAgICAgICAgICAgIDxnIGlkPSJwYXRoMzEzNyI+CiAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTAuMjk2NjI1MzY3LDEzLjE5MDQ4MzEgQzAuMjk2NjI1MzY3LDEwLjM5MTk0ODQgMS4yMDgyNjQ5Myw3LjgwNzU0OTY3IDIuNzQ5ODE2Miw1LjcxNDQ3MDY3IEMzLjIzODg2Mzk2LDUuNzAxOTc0NjggMy42NDg5NTk3Nyw1LjgxMTU5ODYzIDMuNjkyMTI3NzUsNi4wNTg2Nzg1MSBDMy43NDA0MDc3Miw2LjI5NzgwNjM5IDMuNzAwMDc5NzQsNi42NzMyNTQyMiAzLjY1NzQ3OTc2LDcuMDQ4MTM0MDQgQzMuNDA1Mjg3ODgsOS41MjUxODA4NiAzLjMxNTU0MzkzLDEyLjE5ODc1NTYgMy40MTA5Njc4OCwxNC44NzgwMTAzIEMyLjc3MDgzMjE5LDE0Ljg1MDE3ODMgMi4xOTYwMTY0NiwxNC44MjEyMTAzIDEuNjQ1NjI0NzIsMTQuNzg4ODM0MyBDMS42NjcyMDg3MSwxNS4yODQxMzAxIDEuNjkzOTA0NywxNS43NTUwMDE5IDEuNzMzNjY0NjgsMTYuMjQyMzQ1NyBDMi4yNzc4MDg0MiwxNi4yOTQwMzM2IDIuODU0ODk2MTUsMTYuMzQwMDQxNiAzLjQ4NTk0Mzg1LDE2LjM5NDAwMTYgQzMuNTE0MzQzODMsMTYuODIwNTY5NCAzLjU0NTAxNTgyLDE3LjIzMzUwNTIgMy41ODEzNjc4LDE3LjY1MDQxNyBDMy44NjUzNjc2NiwyMC4yMDU4NDc4IDYuNTU3MTE4MzgsMjEuNzY4OTgzIDguNzk4NDQ1MzEsMjIuMzcyMTk4NyBDOS43OTMwMTI4NCwyMi42MTQ3MzQ2IDEwLjQwMTkwODUsMjIuNjkxNDE0NiAxMC44Nzc4OTIzLDIyLjcxNTgzODYgQzExLjQ4Mzk0OCwyMi43Mzg1NTg2IDEyLjIzOTM4NzcsMjIuNzY3NTI2NSAxMi42NTgwMDM1LDIyLjQ4MTgyMjcgQzEzLjE0MDgwMzIsMjIuMTE5NDM4OCAxMy42MTkwNTksMjEuNzYwNDYzIDE0LjEyNjI4MjgsMjEuMzU5NDU1MiBDMTQuMTc5MTA2NywyMS4yOTEyOTUyIDE0LjIzMzYzNDcsMjEuMTgyODA3MyAxNC4xMDY5NzA4LDIwLjk5NDc5OTQgQzEzLjg3Mjk1NDksMjAuNzIzODYzNSAxMy42NDg1OTUsMjAuNDYyMDE1NiAxMy40MTA2MDMxLDIwLjE4MzEyNzggQzEzLjI3NDI4MzIsMjAuMDE2NzAzOSAxMy4wNDgyMTkzLDIwLjA2MTAwNzggMTIuODU2ODAzNCwyMC4xMTI2OTU4IEMxMi40NTg2MzU2LDIwLjMxNTQ3MTcgMTIuMDcyMzk1OCwyMC41MDM0Nzk2IDExLjY4MjE3OTksMjAuNjkzNzU5NSBDMTAuOTc2MTU2MywyMS4wMDg0MzE0IDEwLjI0NTE0MDYsMjEuMzMxMDU1MiA5LjQyOTQ5MzAxLDIxLjI5OTI0NzIgQzguOTIyMjY5MjUsMjEuMjU5NDg3MyA4LjQzMDM4MTQ5LDIwLjczOTE5OTUgOC4zNzc1NTc1MSwyMC4wMjgwNjM4IEM4LjMwODI2MTU1LDE4Ljk0NjU5MjQgOC4yNTk5ODE1NywxNy44MzcyODg5IDguMjIxMzU3NTksMTYuNjMzNjk3NSBDOS42MzM0MDQ5MSwxNi42NzUxNjE0IDExLjA0MjA0NDIsMTYuNjk4NDQ5NCAxMi41MjY3OTU1LDE2LjcwNTI2NTQgQzEyLjUyNjc5NTUsMTYuMzY3ODczNiAxMi41MjM5NTU1LDE2LjA1NzE3NzcgMTIuNTI0NTIzNSwxNS43MTE4MzM5IEMxMi4wNDExNTU4LDE1LjQ5NDg1OCAxMS41ODI3OCwxNS4yNzU2MTAxIDExLjA5ODI3NjIsMTUuMDU0NjU4MiBDMTAuMDk2MzI0NywxNS4wNDIxNjIyIDkuMTQyNjUzMTUsMTUuMDM2NDgyMiA4LjE4MzMwMTYxLDE1LjAxODg3NDIgQzguMTM3ODYxNjMsMTIuMTU5NTYzNiA4LjE4MTU5NzYxLDkuMzA5OTA4OTYgOC4zMDg4Mjk1NSw2LjY3NjA5NDIxIEM4LjMzMjExNzUzLDYuMjc4NDk0NCA4LjM1MzcwMTUyLDUuODc2MzUwNiA4LjQ2ODQzNzQ3LDUuNjA1OTgyNzIgQzguNjA0NzU3NCw1LjI2NzQ1NDg5IDkuMzkzNzA5MDMsNS4wMDY3NDMwMSAxMS4xMDczNjQyLDQuOTA5NjE1MDYgQzExLjExNDc0ODIsNC42MTE5ODMyIDExLjEyMjEzMjIsNC4zMjkxMTkzMyAxMS4xMzIzNTYyLDQuMDQwMDA3NDcgQzguMjc3NTg5NTYsNC4xMjM1MDM0MyA1LjY2OTkwMjgsNC4zODQ3ODMzMSAzLjUxNzc1MTgzLDQuNzcxMDIzMTIgQzUuODI4Mzc0NzMsMi4xOTQwMDgzNSA5LjE4Mjk4MTEzLDAuNTcxODAxMTI2IDEyLjkxNTMwNzMsMC41NzE4MDExMjYgQzE5Ljg4NTgsMC41NzE4MDExMjYgMjUuNTM1MTI1Myw2LjIyMjI2MjQzIDI1LjUzNTEyNTMsMTMuMTkwNDgzMSBDMjUuNTM1MTI1MywxNS44MzYyMjU4IDI0LjcyMTE4MTcsMTguMjkwNTUyNyAyMy4zMjkwMTQ0LDIwLjMxODMxMTcgQzIzLjE1NTIwNjUsMTkuNjQ2MzY4IDIzLjA5NzI3MDUsMTkuMDUyODA4MyAyMi45MzkzNjY2LDE4LjMzNTk5MjcgQzIxLjA0NjIyMzUsMTguNTgxOTM2NSAxOC44MjM2NDA1LDE4Ljc2MjU2MDQgMTYuMzI1NTc3NywxOC44NDk0NjQ0IEMxNi4xMjY3Nzc4LDE4LjIxNTU3NjcgMTUuOTM1MzYxOSwxNy41OTU4ODkgMTUuNzE4Mzg2LDE2LjkxNzY5NzMgQzE5Ljg5NzE2LDE2LjY2MjA5NzUgMjEuOTI4MzI3LDE1LjU2MDc0NiAyMy4xMzE5MTg1LDE0LjI2MzQzNDYgQzI0LjMwNDgzNzksMTIuOTUxMzU1MiAyNC41NTE5MTc4LDExLjUwMDY4MzkgMjQuNDg0ODkzOCwxMC4xNTUwOTI2IEMyNC4zMzU1MDk5LDcuOTY2NTg5NiAyMy4zNzA0Nzg0LDYuNTgyOTQyMjYgMjIuMDQ1OTAzLDUuNTU4ODM4NzUgQzIwLjcwNjU1OTYsNC41NTAwNzEyMyAxOS4wODQ5MjA0LDQuMDQ3OTU5NDcgMTcuNzkwNDQ5LDMuOTM4MzM1NTIgQzE1Ljg3NjI4OTksMy43OTUxOTk1OSAxMy4zMTkxNTUyLDQuMjUwMTY3MzcgMTIuNzc1MDExNCw0LjU4NDE1MTIxIEMxMi40NDk1NDc2LDUuMzMxMDcwODYgMTIuMTI0MDgzNyw2LjA4MDI2MjUgMTEuNzc3NjAzOSw2LjkwMDQ1NDExIEMxMS42OTI5NzE5LDcuMDc0MjYyMDIgMTEuNjM3ODc2LDcuMjU0MzE3OTQgMTEuODc4NzA3OCw3LjQ2MTYzNzg0IEMxMi4yNTAxNzk3LDcuODE3NzczNjcgMTIuNjA4NTg3NSw4LjE2MTk4MTUxIDEyLjk4NTE3MTMsOC41Mjg5MDkzMyBDMTMuMTUwNDU5Miw4LjY5MTM1NzI1IDEzLjMzNjE5NTEsOC44NDI0NDUxOCAxMy40NzUzNTUxLDguNjA4OTk3MjkgQzEzLjk4OTk2MjgsNy44MTY2Mzc2NyAxNC40NzE2MjY2LDcuMDg3ODk0MDIgMTQuOTUzODU4NCw2LjM2MTQyMjM2IEMxNS41MDI1NDYxLDUuNTUzMTU4NzUgMTYuMzM5MjA5Nyw0LjgxNjQ2MzEgMTguMTYzNjI0OCw0LjkzOTcxOTA0IEMxOS42OTg5MjgxLDUuMDc1NDcwOTggMjAuODY1NTk5Niw2LjIzNzU5ODQyIDIwLjk4OTQyMzUsNy41MTM4OTM4MSBDMjEuMTIyOTAzNCw5Ljg3NTA2ODY5IDE4LjczMzg5NjYsMTEuNTUwMDk5OSAxMi41MTY1NzE1LDEyLjA4Njg1OTYgQzEzLjcwOTkzOSwxNS42ODkxMTM5IDE0Ljg3MTQ5ODQsMTkuMTczNzkyMyAxNS44MDI0NSwyMS45NDQ0OTQ5IEMxNy4yNzgxMTMzLDIxLjg3NTE5OSAxOC42NDQ3MjA2LDIxLjc1NTM1MSAxOS45NTA1NTIsMjEuNTg2MDg3MSBDMjAuOTY4OTc1NSwyMS40NTc3MTkyIDIxLjc3NDM5OTEsMjEuMjI2NTQzMyAyMi43MjI5NTg3LDIxLjEzMDU1MTMgQzIwLjQxMDA2MzgsMjMuOTg1MzE4IDE2Ljg3NjUzNzUsMjUuODA5NzMzMSAxMi45MTUzMDczLDI1LjgwOTczMzEgQzUuOTQ3MDg2NjcsMjUuODA5NzMzMSAwLjI5NjYyNTM2NywyMC4xNTk4Mzk4IDAuMjk2NjI1MzY3LDEzLjE5MDQ4MzEiIGlkPSJwYXRoLTEiPjwvcGF0aD4KICAgICAgICAgICAgICAgIDwvZz4KICAgICAgICAgICAgPC9nPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+
```

Comment (zstix) [reply dashboards/f5/f5-big-ip.json:22]: Oh we could simply use the logo stored on the newrelic.com domain:

https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg

Comment (thezackm) [reply dashboards/f5/f5-big-ip.json]: updated; thanks

File: dashboards/f5/f5-big-ip.json [hunk 2]
```diff
@@ -0,0 +1,1184 @@
...
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Nodes\nA [node](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/5.html) is a logical object on the BIG-IP® Local Traffic Manager™ system that identifies the IP address of a physical resource on the network. You can explicitly create a node, or you can instruct Local Traffic Manager (LTM®) to automatically create one when you add a pool member to a load balancing pool.\n\nThe difference between a node and a pool member is that a node is designated by the device’s IP address only (`10.10.10.10`), while designation of a pool member includes an IP address and a service (such as `10.10.10:8`)."
```
Comment (MichelLosier) [inline dashboards/f5/f5-big-ip.json:238]: Lets also update this url to a new relic managed one
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg

File: dashboards/f5/f5-big-ip.json [hunk 3]
```diff
@@ -0,0 +1,1184 @@
...
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Pools\nA [pool](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/6.html#conceptid) is a logical set of devices, such as web servers, that you group together to receive and process traffic. Instead of sending client traffic to the destination IP address specified in the client request, Local Traffic Manager™ sends the request to any of the servers that are members of that pool. This helps to efficiently distribute the load on your server resources."
```
Comment (MichelLosier) [inline dashboards/f5/f5-big-ip.json:456]: Lets update this one as well:
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg

File: dashboards/f5/f5-big-ip.json [hunk 4]
```diff
@@ -0,0 +1,1184 @@
...
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Pool Members\nA [pool member](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/6.html#conceptid) is a logical object that represents a physical node (server), on the network. You then associate the pool with a virtual server on the BIG-IP® system. Once you have assigned a pool to a virtual server, Local Traffic Manager (LTM®) directs traffic coming into the virtual server to a member of that pool. An individual pool member can belong to one or multiple pools, depending on how you want to manage your network traffic."
```
Comment (MichelLosier) [inline dashboards/f5/f5-big-ip.json:648]: Lets update this url:
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg

File: dashboards/f5/f5-big-ip.json [hunk 5]
```diff
@@ -0,0 +1,1184 @@
...
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Virtual Servers\nA [virtual server](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/3.html#conceptid) is a traffic-management object on the BIG-IP system that is represented by an IP address and a service. Clients on an external network can send application traffic to a virtual server, which then directs the traffic according to your configuration instructions. Virtual servers typically direct traffic to a pool of servers on an internal network, by translating the destination IP address in each packet to a pool member address."
```
Comment (MichelLosier) [inline dashboards/f5/f5-big-ip.json:854]: Lets update this url:
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg

**Review by MichelLosier — CHANGES_REQUESTED** [review]

**Pull request merged**